fsharp-backend/tests/FuzzTests/Json.FuzzTests.fs

```diff
@@ -149,10 +151,11 @@ module LibJwtJson =
 
     actual .=. expected
 
-  let roundtripV1 (dv : RT.Dval) : bool =
+
+  let roundtripV0 (dv : RT.Dval) : bool =
```

Comment:
Here I simply moved the v0 fn above the v1 fn, to be consistent with the test list at the end of the module - looks like more churn than it is.